sys/ztimer/auto_init.c

```diff
@@ -250,7 +279,31 @@ void ztimer_init(void)
 #  endif
 #endif
 
+#if INIT_ZTIMER_PTP
+    LOG_DEBUG("ztimer_init(): initializing PTP\n");
+    ztimer_periph_ptp_init(&ZTIMER_PTP);
+#  ifdef MODULE_PM_LAYERED
+     LOG_DEBUG("ztimer_init(): ZTIMER_PTP setting block_pm_mode to %i\n",
+               CONFIG_ZTIMER_PTP_BLOCK_PM_MODE);
+     ZTIMER_PTP_CLK.block_pm_mode = CONFIG_ZTIMER_PTP_BLOCK_PM_MODE;
+#  endif
```

Comment:
crazy uncrustify doesn't like that indention. I am not sure if the suggestion i is good